.evergreen/config.yml

```diff
@@ -295,7 +295,7 @@ buildvariants:
 
   - name: oidc
     display_name: OIDC
-    patchable: false
```

Comment:
Please remember to switch this back before merging (speaking from experience, it's easy to forget)